serving/fn/openapi/PatientVitals.java

```diff
@@ -0,0 +1,554 @@
+
+package com.redhat.demo.sepsis.beans;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+
+/**
+ * Root Type for PatientVitals
+ * <p>
+ * Vitals information taken from a patient at a particular point in time.
+ * 
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+    "HR",
+    "O2Sat",
+    "Temp",
+    "SBP",
+    "MAP",
+    "DBP",
+    "Resp",
+    "EtCO2",
+    "BaseExcess",
+    "HCO3",
+    "FiO2",
+    "pH",
+    "PaCO2",
+    "SaO2",
+    "AST",
+    "BUN",
+    "Alkalinephos",
+    "Calcium",
+    "Chloride",
+    "Creatinine",
+    "Bilirubin_direct",
+    "Glucose",
+    "Lactate",
+    "Magnesium",
+    "Phosphate",
+    "Potassium",
+    "Bilirubin_total",
+    "TroponinI",
+    "Hct",
+    "Hgb",
+    "PTT",
+    "WBC",
+    "Fibrinogen",
+    "Platelets",
+    "Age",
+    "Gender",
+    "Unit1",
+    "Unit2",
+    "HospAdmTime",
+    "ICULOS",
+    "isSepsis"
+})
+public class PatientVitals {
```

Comment:
because the pipeline.pkl scales the data and applies a median for missing values, can you change the 'string' type to float64. the only other data type from the source is int64, but would get dropped from FIHR or in the model_process.py class

Reply:
i've asked @dvanbalen to update the openapi spec to change all data types from string to number (which should generate a float in the client java stub).

Reply:
code changed

Reply:
pushed the new api spec and subs